normalize.css

```diff
@@ -0,0 +1,351 @@
+/*! normalize.css v8.0.1 | MIT License | github.com/necolas/normalize.css */
+
+/* Document
+   ========================================================================== */
+
+/**
+ * 1. Correct the line height in all browsers.
+ * 2. Prevent adjustments of font size after orientation changes in iOS.
+ */
+
```

Comment:
- [ ] Please delete commented code.
<img width="792" alt="no_inline_comments" src="https://user-images.githubusercontent.com/51926937/76787422-9fa2ac80-677e-11ea-9b10-faa8a491b7f7.png">

There is no need to include such long comments since your code should be descriptive enough. You can add some divisors if you need to keep track of your CSS sections.

Please make all the necessary changes in your CSS files.